src/routes/formsgSiteCreation.ts

```diff
@@ -157,9 +167,83 @@ export class FormsgRouter {
     await mailer.sendMail(email, subject, html)
   }
 
+  cloneSiteToEfs: RequestHandler<
+    never,
+    Record<string, never>,
+    { data: { submissionId: string } },
+    never,
+    { submission: DecryptedContent }
+  > = async (req, res) => {
+    // 1. Extract arguments
+    const { submissionId } = req.body.data
+    const { responses } = res.locals.submission
+    // NOTE: This is validated by formsg to be of domain `@open.gov.sg`;
+    // hence, not revalidating here
```

Comment:
Hmm lets revalidate here endsWith("@open.gov.sg")? Since this piece of infra (ie formsg) is not documented and subject to change, it is a quick win to revalidate here! 
PS. there was an vulnerability before where the form fields could have been mutated maliciously, rather stick to best practices here to revalidate!

Reply:
sure, sgtm